2.7/Dockerfile

```diff
@@ -44,16 +44,16 @@ RUN set -ex \
 	&& make install \
 	&& ldconfig \
 	\
-		&& wget -O /tmp/get-pip.py 'https://bootstrap.pypa.io/get-pip.py' \
-		&& python2 /tmp/get-pip.py "pip==$PYTHON_PIP_VERSION" \
-		&& rm /tmp/get-pip.py \
+	&& wget -O /tmp/get-pip.py 'https://bootstrap.pypa.io/get-pip.py' \
+	&& python2 /tmp/get-pip.py "pip==$PYTHON_PIP_VERSION" \
+	&& rm /tmp/get-pip.py \
```

Comment:
This looks like it was unintentionally added here -- these are kept indented so that diff between this version and the other versions is smaller (no whitespace changes, only removed lines, essentially).